yellowbrick/classifier/classification_report.py

```diff
@@ -122,17 +136,17 @@ def draw(self):
         Renders the classification report across each axis.
         """
         # Create display grid
-        cr_display = np.zeros((len(self.classes_), 4))
+        cr_display = np.zeros((len(self.classes_), len(self.measurements)))
 
 
         # For each class row, append columns for precision, recall, f1, and support
         for idx, cls in enumerate(self.classes_):
-            for jdx, metric in enumerate(('precision', 'recall', 'f1', 'support')):
+            for jdx, metric in enumerate(self.measurements):
                 cr_display[idx, jdx] = self.scores_[metric][cls]
```

Comment:
Slight bug injected here from my above code proposals - for `support`, this value must always be the percentage,  and not the count if that's the values stored, this was guaranteed before since support was always a percentage in the `scores_` dictionaries. So depending on how you proceed with this, I just wanted to highlight this minor issue.

Reply:
A big question for me...Where do I store the support counts?  Since, percent needs to be store in ```scores_```